docs/maven-example.md

```diff
@@ -0,0 +1,56 @@
+# Fabric8 Maven Plugin + Kompose:
+
+Lets deploy a Springboot Java application with Docker Compose file using fabric8 maven plugin to Kubernetes/Openshift.
+
+__1. Clone the example project from github__
+```bash
+$ git clone https://github.com/piyush1594/Kompose-Example.git
```

Comment:
nitpick, but would be better having `Kompose-Example` lowercase, could you change the name of your repo?